cmd/minikube/cmd/dashboard.go

```diff
@@ -53,19 +54,24 @@ var dashboardCmd = &cobra.Command{
 			cmdutil.MaybeReportErrorAndExit(err)
 		}
 
-		url, err := cluster.GetServiceURL(api, namespace, service, nil)
+		urls, err := cluster.GetServiceURLs(api, namespace, service, nil)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			fmt.Fprintln(os.Stderr, "Check that minikube is running.")
 			cmdutil.MaybeReportErrorAndExit(err)
 		}
+		if len(urls) == 0 {
+			errMsg := "There appears to be no url associated with dashboard, this is not expected, exiting"
+			glog.Infoln(errMsg)
+			fmt.Println(errMsg)
```

Comment:
I did this so that it would display to the user as well as show up in the logs.  This way the user does not have the `minikube dashboard` command exit for seemingly no reason.  I can change it to only log to stderr though, would that be better?